lib/routes/user/marketplace/vendor_webview.dart

```diff
@@ -64,10 +64,80 @@ class VendorWebViewPageState extends State<VendorWebViewPage> {
     });
   }
 
+  _injectWebLNLibrary() {
+    String includeWebLN = "var webLNLib = document.createElement('script');\n" +
+        "webLNLib.src = 'https://unpkg.com/webln@0.2.0/dist/webln.min.js';\n" +
+        "webLNLib.integrity = 'sha384-mTReBqbhPO7ljQeIoFaD1NYS2KiYMwFJhUNpdwLj+VIuhhjvHQlZ1XpwzAvd93nQ';\n" +
+        "webLNLib.crossOrigin = 'anonymous';\n" +
+        "document.body.appendChild(webLNLib);";
+    _widgetWebview.evalJavascript(includeWebLN).whenComplete(() => _initializeWebLN());
+  }
+
+  _initializeWebLN() {
+    String initWebLN = "   var processedInvoices = [];\n" +
+        "       webln = {\n" +
+        "   enable: function () {\n" +
+        "       window.postMessage(JSON.stringify({ enable: true }));\n" +
+        "       return new Promise(function (resolve, reject) {resolve(true);});\n" +
+        "   },\n" +
+        "   sendPayment: function (paymentRequest) {\n" +
+        "       window.postMessage(JSON.stringify({ uri: paymentRequest }));\n" +
+        "       processedInvoices.push(paymentRequest);\n" +
+        "       return new Promise(function (resolve, reject) { });\n" +
+        "   },\n" +
+        "};\n" +
+        "setTimeout(function () { WebLN.requestProvider().catch(e => console.log(e));},450);";
+    _widgetWebview.evalJavascript(initWebLN).whenComplete(() => _findBolt11Links());
+  }
+
+  _findBolt11Links() {
```

Comment:
This is not in webln standard. I don't think we should add this dynamic search for links here. Let's stick to webln interface.